source/community/reactnative/ios/RNAdaptiveCards.xcodeproj/xcshareddata/xcschemes/RNAdaptiveCards-tvOS.xcscheme

```diff
@@ -1,25 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0940"
+   LastUpgradeVersion = "1130"
    version = "1.3">
    <BuildAction
-      parallelizeBuildables = "NO"
+      parallelizeBuildables = "YES"
```

Comment:
shouldn't this be "No"

Reply:
For the tv-OS scheme - "parallelizeBuildables" should be YES.